
--- src/App.tsx
+++ src/App.tsx
@@ -22,6 +22,7 @@ import SupplierNew from './pages/fornecedores/SupplierNew';
 import SupplierEdit from './pages/fornecedores/SupplierEdit';
 import SupplierDetail from './pages/fornecedores/SupplierDetail';
 
+import TransactionList from './pages/transacoes/TransactionList';
 import CategoryList from './pages/categorias/CategoryList';
 import CategoryNew from './pages/categorias/CategoryNew';
 import CategoryEdit from './pages/categorias/CategoryEdit';
@@ -64,6 +65,7 @@ function App() {
               <Route path="/produtos/editar/:id" element={<ProductEdit />} />
               <Route path="/produtos/:id" element={<ProductDetail />} />
               <Route path="/dashboard" element={<DashboardPage />} />
+              <Route path="/transacoes" element={<TransactionList />} />
               <Route path="/categorias" element={<CategoryList />} />
               <Route path="/categorias/nova" element={<CategoryNew />} />
               <Route path="/categorias/editar/:id" element={<CategoryEdit />} />
